bluemira/codes/process/mapping.py

```diff
@@ -86,6 +135,13 @@
     "r_vv_ob_in": ("r_vv_ob_in", "m"),
     "r_tf_out_centre": ("r_tf_outboard_mid", "m"),
     "g_vv_ts": ("gapds", "m"),
+    "TF_res_bus": ("tfbusres", "m"),
+    "TF_res_tot": ("ztot", "ohm"),
+    "TF_E_tot": ("estotftgj", "GJ"),
+    "TF_respc_ob": ("tflegres", "ohm"),
+    "TF_currpt_ob": ("cpttf", "A"),
+    "P_bd_in": ("pinjmw", "MW"),
+    "condrad_cryo_heat": ("qss/1.0D6", "MW"),
```

Comment:
Not sure about the others, shouldn't these go in config.py too? Would help to have full parameters for them

Reply:
yeah and config_schema.py, I just wanted to have the names sorted before I did that